src/cache.js

```diff
@@ -69,6 +69,25 @@ query getProjectsV2Fields($id: ID!, $cursor: String!) {
 	}
 }
 `;
+const GetRepositoryDescription = gql`
+query GetRepositoryDescription($orgName: String!, $repoName: String!){
+    repository(owner: $orgName, name: $repoName) {
+		description
+        main: object(expression: "main:README.md") {
+            ... on Blob {
+                text
+            }
+        }
+        master: object(expression: "master:README.md") {
+            ... on Blob {
+                text
+            }
+        }        
+    }
+}
+`; //Loophole to get the ReadMe No Matter the branch name
```

Comment:
The website is behind on this, but we've been moving to the comment format made by [better comments](https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments)

```suggestion
`; // * Loophole to get the ReadMe No Matter the branch name
```